packages/jest-editor-support/src/Process.js

```diff
@@ -27,16 +28,26 @@ export const createProcess = (
   // any other bits into the args
   const runtimeExecutable = workspace.pathToJest;
   const parameters = runtimeExecutable.split(' ');
-  const command = parameters[0];
   const initialArgs = parameters.slice(1);
-  const runtimeArgs = [].concat(initialArgs, args);
+  let command = parameters[0];
+  let runtimeArgs = [].concat(initialArgs, args);
 
   // If a path to configuration file was defined, push it to runtimeArgs
   if (workspace.pathToConfig) {
     runtimeArgs.push('--config');
     runtimeArgs.push(workspace.pathToConfig);
   }
 
+  if (workspace.useWsl) {
+    // useWsl can be either true for the default ('wsl' or the explicit
+    // wsl call to use, e.g. 'ubuntu run')
+    const wslCommand = workspace.useWsl === true ? 'wsl' : workspace.useWsl;
```

Comment:
In order to not duplicate your code, I would use your `getWslCommand` function from readTestResults.js (even though I would then place the function in another file, maybe we need a helper.js file or similar).